src/components/TF2/Player/Player.tsx

```diff
@@ -189,6 +190,11 @@ const Player = ({
             // Immediately update local instance
             // Causes new info to immediately show
             player.localVerdict = e.toString();
+            color = displayColor(playerColors!, player, cheatersInLobby);
+            const newColour = color ? `background-color: ${color}` : '';
+            document
+              .getElementById(`player-display-div-${player.steamID64}`)
+              ?.setAttribute('style', newColour);

```

Comment:
setAttribute overwrites all style attributes associated with the element; which could break a lot of stuff.

setProperty('background-color", newColor) would be the more appropriate function; assuming the rest of the code stayed the same.

Reply:
setAttribute was used because the divs never have attributes unless they have a colour, for context